pkg/agent/openflow/pipeline.go

```diff
@@ -207,6 +207,7 @@ const (
 	serviceLearnReg         = endpointPortReg // Use reg4[16..18] to store endpoint selection states.
 	EgressReg       regType = 5
 	IngressReg      regType = 6
+	DispositionReg  regType = 7 // Stores Allow or Drop
```

Comment:
If there are only 2 candidate values, using a complete register(32 bits) is a waste. Is it possible to use 1 or 2 bits in some shared register?

Reply:
May I ask which register would be available for sharing at this time? Thanks.

Reply:
Is it possible to leverage the 21th bit of reg0? If you want to have some buffer to extend candidate value, you could leverage 21..31 bits of reg0.